stdlib/testing/diff.go

```diff
@@ -504,8 +504,9 @@ func (t *DiffTransformation) rowEqual(want, got *tableBuffer, i int) bool {
 		switch wantCol.Type {
 		case flux.TFloat:
 			want, got := wantCol.Values.(*array.Float64).Value(i), gotCol.Values.(*array.Float64).Value(i)
-			// want == got is for handling +Inf and -Inf.
-			return want == got || math.Abs(want-got) <= t.epsilon
+			if math.Abs(want-got) > t.epsilon {
```

Comment:
This looks different. The original says that `want == got` was for handling `+Inf` and `-Inf`. If we remove that section of code, do we still handle that case?

Reply:
We do have a couple of existing tests for +/-Inf and it's all passing with my change. https://github.com/influxdata/flux/blob/master/stdlib/testing/diff_test.go#L233

But I agree we should have some robust handling of Inf case for all the possibilities. 

FYI, go behavior I tried for Inf - 

```
fmt.Printf("epsilon  = %v\n", math.Abs(math.Inf(1) - math.Inf(2))) // NaN
fmt.Printf("epsilon  = %v\n", math.Inf(1) == math.Inf(2)) // true
fmt.Printf("%v", math.IsInf(math.Inf(+1), 0)) // true
```

I will update the diff with my new checks.